test/metabase/upload_test.clj

```diff
@@ -2391,7 +2426,7 @@
 
 (deftest unique-long-column-names-test
   (let [original ["αbcdεf_αbcdεf"     "αbcdεfg_αbcdεf"   "αbc_2_etc_αbcdεf" "αbc_3_xyz_αbcdεf"]
-        expected [:%CE%B1bcd%  :%_852c229f :%CE%B1bc_2 :%CE%B1bc_3]
+        expected [:%ce%b1bcd%  :%_b59bccce :%ce%b1bc_2 :%ce%b1bc_3]
```

Comment:
The change to the second field might be a blocker 😢 

Might need another way to get around default case differences between databases.

Reply:
We decided its OK to break appends to existing files with duplicated non-ascii headers. Appending didn't work without this change, and maintaining backwards compatibility with those existing columns would introduce a lot of complexity.